docs/implement-api-bindings.md

```diff
@@ -11,119 +10,88 @@ Sections:
 - the URL path selects the command - e.g `/object/get`
 - the URL query string implements option arguments - e.g. `&enc=protobuf&q=true`
-- the URL query also implements positional arguments - e.g.
-  `&arg=<hash1>&arg=add-link&arg=foo&arg=<hash2>`
+- the URL query also implements positional arguments - e.g. `&arg=<hash1>&arg=add-link&arg=foo&arg=<hash2>`
 - the request body streams file data - reads files or stdin
   - multiple streams are muxed with multipart (todo: add tar stream support)
 
 To date, we have two different HTTP API clients:
 
-- [js-ipfs-api](https://github.com/ipfs/js-ipfs-api) - simple javascript
-  wrapper -- best to look at
-- [go-ipfs/commands/http](https://git.io/v5KnB) -
-  generalized transport based on the [command definitions](https://git.io/v5KnE)
+- [js-ipfs-api](https://github.com/ipfs/js-ipfs-api) - simple javascript wrapper -- best to look at
+- [go-ipfs/commands/http](https://git.io/v5KnB) - generalized transport based on the [command definitions](https://git.io/v5KnE)
 
-The Go implementation is good to answer harder questions, like how is multipart
-handled, or what headers should be set in edge conditions. But the javascript
-implementation is very concise, and easy to follow.
+The Go implementation is good to answer harder questions, like how is multipart handled, or what headers should be set in edge conditions. But the javascript implementation is very concise and easy to follow.
 
 #### Anatomy of node-ipfs-api
 
 Currently, node-ipfs-api has three main files
-- [src/index.js](https://git.io/v5Kn2) defines the functions clients of the API
-  module will use. uses `RequestAPI`, and translates function call parameters to
-  the API almost directly.
-- [src/get-files-stream.js](https://git.io/v5Knr) implements the hardest part:
-  file streaming. This one uses multipart.
-- [src/request-api.js](https://git.io/v5KnP) generic function call to perform
-  the actual HTTP requests
+- [src/index.js](https://git.io/v5Kn2) defines the functions clients of the API module will use. uses `RequestAPI`, and translates function call parameters to the API almost directly.
+- [src/get-files-stream.js](https://git.io/v5Knr) implements the hardest part: file streaming. This one uses multipart.
+- [src/request-api.js](https://git.io/v5KnP) generic function call to perform the actual HTTP requests
 
 ## Note on multipart + inspecting requests
 
-Despite all the generalization spoken about above, the IPFS API is actually very
-simple. You can inspect all the requests made with `nc` and the `--api` option
-(as of [this PR](https://github.com/ipfs/go-ipfs/pull/1598), or `0.3.8`):
+Despite all the generalization spoken about above, the IPFS API is actually very simple. You can inspect all the requests made with `nc` and the `--api` option (as of [this PR](https://github.com/ipfs/go-ipfs/pull/1598), or `0.3.8`):
 
-```
-> nc -l 5002 &
-> ipfs --api /ip4/127.0.0.1/tcp/5002 swarm addrs local --enc=json
+```console
+nc -l 5002 &
```

Comment:
Let's keep the `> ` (and use it everywhere). Otherwise, it's impossible to distinguish between input and output.

Reply:
how about $? Since we use bash that's actually more close to the real shell. The ArchLinux uses this for example :)

Reply:
I agree `$` would make more sense, but we use `>` almost everywhere (there was a good reason to pick it, IIRC, but I can't remember where that discussion happened). Changing this isn't worth it.